src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -298,7 +298,11 @@ bool Dataloader::load_defines(GameManager& game_manager) const {
 		Logger::error("Failed to load goods!");
 		ret = false;
 	}
-	if (!_load_pop_types(game_manager.get_pop_manager(), pop_type_directory)) {
+	if (!_load_units(game_manager, units_directory)) {
+		Logger::error("Failed to load units!");
+		ret = false;
+	}
+	if (!_load_pop_types(game_manager.get_pop_manager(), game_manager, pop_type_directory)) {
```

Comment:
Probably be better to leave reorganizing the position of `_load_units` to another PR, (a lot of reorganization is gonna be needed soon anyway, like running formatting and include-what-you-use) honestly we should really start moving to a "minimal purpose built changes" model for PRs, but that can be discussed in dev chat, for now the `_load_units` move should be moved to a separate PR.

Reply:
The reorganisation isn't arbitrary/separate to the main purpose of the PR, it's necessary to be able to look up unit types while parsing pop types.